GVFS/GVFS.Common/GVFSPlatform.cs

```diff
@@ -88,6 +88,8 @@ public bool TryGetNormalizedPathRoot(string path, out string pathRoot, out strin
             return true;
         }
 
+        public abstract string GetProgramLocater();
```

Comment:
Feedback from #1129 - make this a property and move into `GVFSPlatformConstants`